cmd/pebble/cmd_replan_test.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2022 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main_test
+
+import (
+	"fmt"
+	pebble "github.com/canonical/pebble/cmd/pebble"
+	"gopkg.in/check.v1"
+	"net/http"
```

Comment:
Nit: we like to keep imports in three separate groups, separated by a blank line: first standard library imports, next third party packages, then local Pebble imports. [Example.](https://github.com/canonical/pebble/blob/46b3a83075bf7329fae3b732b322a56b01a328c7/cmd/pebble/cmd_plan_test.go#L17-L25)